packages/ts-bundler/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "@reapit/ts-bundler",
+  "main": "src/ts-bundler.ts",
+  "private": true,
```

Comment:
I think you might need to add the normal build, test etc commands but echoed out - not 100% but think some of the GH workflows might fail otherwise

Reply:
good catch!